phpunit.xml

```diff
@@ -10,22 +10,24 @@
     convertWarningsToExceptions="true"
```

Comment:
a few of these are the same as their default values. But left them alone as this PR is just about updating the config